### ISSUE TITLE
win binaries pls )

### ISSUE THREAD

**Issue by necros2k7** [gurka/OpenCrystalCaves#5]
subj

**Comment by JayFoxRox** [comment]
I feel like this issue could have been written more professionally. Aside from these style issues in the request there are some other political and logic issues:

- You'd still need to run the Python script to extract the sprites.
- You'd still need to manually fetch the font file.
- You still couldn't play the game due to shortcomings in the implementation (See README: no enemies, no shooting, no entering levels).
- You can't fix those issues despite FOSS because you probably don't have the toolchain set up to immediately start working.

The typical motivation for using FOSS is that you can fix issues yourself, or port it to other platforms; if you don't care about that, then you can also just use the original Windows release from Steam (which you need regardless). I don't see why you'd care for a Windows binary?

Compiling this project is also trivial and I'd encourage any developer (assuming you are one, given that you use GitHub) to just compile it themselves (for security but also for being able to help with projects).

---

If binaries were to become a reality, it should happen like this:

- Bundle the font into the media folder (which the license allows).
- Replace the sprite-extractor with C code which automatically loads it.
- Push artifact via some trusted CI system (to avoid malware from a developers machine infecting a release).

Regardless, this would just be future-proofing for when you'd actually benefit from having a playable binary. Considering the [last update in this codebase has been more than 2 years ago](https://github.com/gurka/OpenCrystalCaves/graphs/code-frequency) it's relatively safe to assume this won't happen anytime soon (although I hope it does happen *eventually*).

**Comment by gurka** [comment]
>Considering the last update in this codebase has been more than 2 years ago it's relatively safe to assume this won't happen anytime soon (although I hope it does happen eventually).

Yep. I have no plans (nor time) at the moment to continue with this project.
